packages/web/utils/dework.ts

```diff
@@ -61,9 +77,23 @@ export const processDeworkData = (data: DeworkData) => {
     });
     // get usdc rewards
     element?.rewards?.forEach(({ token, amount }) => {
-      if (usdcTokens.includes(token.address || 'empty')) return;
-      if (amount) {
-        totalEarnedInUSDC += +amount;
+      if (
+        usdcTokens6Decimals.includes(token.address?.toLowerCase() || 'empty') &&
```

Comment:
This *should* work, but it's not the most straightforward statement.

```suggestion
      !!token.address &&
       
usdcTokens6Decimals.includes(token.address.toLowerCase())
```
Says more explicitly what is being checked.